Description: Spelling fixes (thanks Paul Wise)
Author: Tias Guns <tias@ulyssis.org>
Forwarded: yes (I am upstream)

--- a/man/xinput_calibrator.1
+++ b/man/xinput_calibrator.1
@@ -46,7 +46,7 @@
 .B \-\-precalib \fImin_x\fP \fImax_x\fP \fImin_y\fP \fImax_y\fP
 Manually provide the current calibration setting.
 .br 
-This is usefull if the calibration values are stored in your xorg.conf, but the driver does not export them through XInput (eg. the calibrator can not know these values)
+This is useful if the calibration values are stored in your xorg.conf, but the driver does not export them through XInput (eg. the calibrator can not know these values)
 .PP 
 .TP 8
 .B \-\-misclick \fInr_of_pixels\fP
@@ -56,7 +56,7 @@
 .B \-\-fake
 Emulate a fake driver (for testing purposes)
 .br 
-Usefull to test the calibrator without applying the values, and possibly even without having a touchscreen.
+Useful to test the calibrator without applying the values, and possibly even without having a touchscreen.
 .SH "USAGE"
 Run xinput_calibrator in a terminal, as it prints out the calibration values and instructions on standard output.
 .PP 
--- a/src/calibrator/calibratorEvdev.cpp
+++ b/src/calibrator/calibratorEvdev.cpp
@@ -133,7 +133,7 @@
 
             if (verbose) {
                 if (ok)
-                    printf("DEBUG: Succesfully applied axis calibration.\n");
+                    printf("DEBUG: Successfully applied axis calibration.\n");
                 else
                     printf("DEBUG: Failed to apply axis calibration.\n");
             }
@@ -197,7 +197,7 @@
 
         if (verbose) {
             if (ok)
-                printf("DEBUG: Succesfully swapped X and Y axis.\n");
+                printf("DEBUG: Successfully swapped X and Y axis.\n");
             else
                 printf("DEBUG: Failed to swap X and Y axis.\n");
         }
@@ -210,7 +210,7 @@
 
     if (verbose) {
         if (ok)
-            printf("DEBUG: Succesfully applied axis calibration.\n");
+            printf("DEBUG: Successfully applied axis calibration.\n");
         else
             printf("DEBUG: Failed to apply axis calibration.\n");
     }
--- a/src/calibrator/calibratorUsbtouchscreen.cpp
+++ b/src/calibrator/calibratorUsbtouchscreen.cpp
@@ -187,7 +187,7 @@
     // to keep the for the next boot
     FILE *fid = fopen(modprobe_conf_local, "r");
     if (fid == NULL) {
-        fprintf(stderr, "Error: Can't open '%s' for reading. Make sure you have the necesary rights\n", modprobe_conf_local);
+        fprintf(stderr, "Error: Can't open '%s' for reading. Make sure you have the necessary rights\n", modprobe_conf_local);
         fprintf(stderr, "New calibration data NOT saved\n");
         return false;
     }
@@ -217,7 +217,7 @@
 
     fid = fopen(modprobe_conf_local, "w");
     if (fid == NULL) {
-        fprintf(stderr, "Error: Can't open '%s' for writing. Make sure you have the necesary rights\n", modprobe_conf_local);
+        fprintf(stderr, "Error: Can't open '%s' for writing. Make sure you have the necessary rights\n", modprobe_conf_local);
         fprintf(stderr, "New calibration data NOT saved\n");
         return false;
     }
